client/config.c

```diff
@@ -326,6 +390,9 @@ TDNFFreeConfig(
         TDNF_SAFE_FREE_MEMORY(pConf->pszVarReleaseVer);
         TDNF_SAFE_FREE_MEMORY(pConf->pszVarBaseArch);
         TDNF_SAFE_FREE_MEMORY(pConf->pszBaseArch);
+        TDNF_SAFE_FREE_MEMORY(pConf->pszUserAgentHeader);
```

Comment:
Can we have a function of it's own to free up pConf structure? this is too much.

Reply:
Any thoughts on this?

Reply:
Ah, this *is* in `TDNFFreeConfig()` already ... I think the UI tricked us here.